.github/workflows/maven.yml

```diff
@@ -112,7 +112,14 @@ jobs:
 
       - name: Run benchmark
         run: |
-          java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/sqlite/sample_${{matrix.benchmark}}_config.xml --create=true --load=true --execute=true --json-histograms results/histograms.json
+          # For templated benchmarks, we need to preload some data for the test since by design, templated benchmarks do not support the 'load' operation
+          # In this case, we load the tpcc data.
+          if [[ ${{matrix.benchmark}} == templated ]]; then
```

Comment:
Looks like I missed listing the benchmark in the matrix above.  Can you add that too please?

Reply:
Trying to see if I can figure out why the docker build was failing next ...

Reply:
Here's the full set of fixes I applied (skipped a PR this time, hope that's OK):
https://github.com/bpkroth/benchbase/compare/681c0aa7..bc96fcad

Basically, there's 
- a few more fixes to the local testing scripts
- an tweak to speed up tpcc data loading (quicker testing)
- a fix to a prior change I made while enabling unit test support for templated benchmarks
- a fix for a bug I hadn't encountered previously where the wrong driver is used to double check the catalog for sqlite
- a fix to the LIMIT syntax used in a test template query (sqlserver uses TOP instead)